test/client/components/script/script_question_list.test.js

```diff
@@ -0,0 +1,30 @@
+import fixtures from '../../client_fixtures';
+import { checkObjectProps } from '../../client_test_helpers.js';
+
+const { scriptQuestionsFixture: questions } = fixtures.scriptViewFixtures;
+
+const props = {
+  questions
+};
+
+describe('QuestionList component', () => {
+  const { questions } = props;
```

Comment:
I think when checking props in component test it might be better to shallowly render the component. Otherwise you're just confirming that the constant "props" you defined it defined.